shellcheck.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+SHELLCHECK_VERSION=0.7.1
+podman run --rm -it -v "$(pwd):/scripts:z" peterdavehello/shellcheck:$SHELLCHECK_VERSION /bin/sh -c "shellcheck -S error /scripts/*.sh"
```

Comment:
` -v "$(pwd):/scripts:z,ro"`, I don't think we need to give write access?

shellcheck is packaged in EPEL, I don't know if there's a way to get it added to the CI images.